azext_iot/monitor/parsers/issue.py

```diff
@@ -17,7 +18,11 @@ def __init__(self, severity: Severity, details: str, message, device_id=""):
         self.severity = severity
         self.details = details
         self.device_id = device_id
-        self.message = str(message)
+        if message:
+            message_body = next(message.get_data())
+            self.message = unicode_decode(data=message_body, default="Failed to represent content in unicode format.")
+        else:
```

Comment:
I'm curious what is the case for when message passed into init here is falsey? i.e. is this else needed? What if we set message=None by default?

Reply:
There isn't a case yet that the message is falsy, I was thinking since we are going to use message.get_data() so just want to make sure it's not falsy in the first place if there is any use case in the future. But you mentioned this issue handler is not been used except central, so I think it should be good to remove the check, thanks!